cluster/images/etcd/Makefile

```diff
@@ -31,10 +31,10 @@ BUNDLED_ETCD_VERSIONS?=3.0.17 3.1.12 3.2.24 3.3.17 3.4.13 3.5.0-beta.3
 # LATEST_ETCD_VERSION identifies the most recent etcd version available.
 LATEST_ETCD_VERSION?=3.5.0-beta.3
 
-# REVISION provides a version number fo this image and all it's bundled
+# REVISION provides a version number for this image and all it's bundled
 # artifacts. It should start at zero for each LATEST_ETCD_VERSION and increment
 # for each revision of this image at that etcd version.
-REVISION?=0
+REVISION?=1
```

Comment:
We did this the last time we bumped images (reference: https://github.com/kubernetes/kubernetes/pull/100976/files#diff-de315dbb88d6d341b1682ef2fb0798723139fb4202ac607a334fc301e9359ec7R37), so I thought we should do it this time too.

Reply:
doesn't hurt usually to do this. however we still haven't updated to the 3.5 beta image yet ( https://github.com/kubernetes/kubernetes/blob/fd82c69957b08ff3fad6aadb6b652bef1490b889/build/dependencies.yaml#L79-L93 ) so we could keep REVISION to `0` . But will defer to the @kubernetes/sig-release-leads

Reply:
It seems like the etcd `3.5.0-beta.3-1` image has been already built, so I believe it doesn't make sense to revert the REVISION to `0` any longer. Ref: https://github.com/kubernetes/k8s.io/pull/2101